plugins/SimulationView/SimulationView.py

```diff
@@ -334,36 +331,51 @@ def getExtruderOpacities(self) -> Matrix:
         return Matrix(self._extruder_opacity)
 
     def setShowTravelMoves(self, show):
```

Comment:
Nitpicking: Boyscouting rules say that you should type this function

Reply:
Indeed. Done!